packages/url_launcher/url_launcher/lib/src/link.dart

```diff
@@ -90,16 +90,17 @@ class DefaultLinkDelegate extends StatelessWidget {
   bool get _useWebView {
     if (link.target == LinkTarget.self) return true;
     if (link.target == LinkTarget.blank) return false;
-    return null;
+    return false;
   }
 
   Future<void> _followLink(BuildContext context) async {
-    if (!link.uri.hasScheme) {
+    if (link.uri == null || !link.uri!.hasScheme) {
```

Comment:
This function is not supposed to be called when `uri` is null. I think it's more appropriate to force it to be non-null:
```dart
if (!link.uri!.hasScheme) {
```

Reply:
Done